### PR TITLE
click "functions" to open symbol search modal

### PR DESCRIPTION
**Pull request by aklt**
Associated Issue: #3233

### Summary of Changes

* Add a link that pops up the functions search modal when clicked to the find text UI.

### Test Plan

* Press CmdOrCtrl + F to bring up the text search dialog
* Click the "Search functions..." text to bring up the functions search modal

### Screenshots/Videos (OPTIONAL)

ffdebug

### CODE REVIEW

File: src/components/Editor/SearchBar.js [hunk 1]
```diff
@@ -378,6 +379,27 @@ class SearchBar extends Component {
...
+        "Search:"
+      ),
+      dom.span(
+        {
+          className: "search-type-name",
+          onClick: () => {
+            this.props.setActiveSearch("symbol");
+          }
+        },
+        "functions"
```
Comment (jasonLaster) [inline src/components/Editor/SearchBar.js]: Lets change this to `Search for functions` and move it to L10N
https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n

**Comment by jasonLaster** [comment]
screen shot 2017-08-10 at 6 20 07 pm


Modifiers and the buttons too (maybe) look a bit big

File: src/components/Editor/SearchBar.js [hunk 2]
```diff
@@ -380,6 +383,21 @@ class SearchBar extends Component {
     );
   }
 
+  renderSearchType() {
+    return (
+      <div className="search-type-toggles">
+        <span
+          className="search-type-name"
+          onClick={() => {
```
Comment (jasonLaster) [inline src/components/Editor/SearchBar.js]: `onClick={() =>  this.props.setActiveSearch("symbol")}`

because, why not

Comment (aklt) [inline src/components/Editor/SearchBar.js]: Do you mean the formatting?

**Pull request merged**